kore/src/Kore/Step/Simplification/Condition.hs

```diff
@@ -71,22 +71,21 @@ unmodified.
 -}
 simplify ::
     forall simplifier any.
-    ( HasCallStack
-    , MonadSimplify simplifier
-    ) =>
+    HasCallStack =>
+    MonadSimplify simplifier =>
     SubstitutionSimplifier simplifier ->
     SideCondition RewritingVariableName ->
     Conditional RewritingVariableName any ->
     LogicT simplifier (Conditional RewritingVariableName any)
 simplify SubstitutionSimplifier{simplifySubstitution} sideCondition =
-    normalize >=> worker
+    normalize >=> worker 0
   where
-    worker Conditional{term, predicate, substitution} = do
+    worker count Conditional{term, predicate, substitution} = do
```

Comment:
Is `count` used anywhere?

Reply:
This is left-over from debugging. I suppose we should add a warning here, the same as in the `Predicate` simplifier.